### DISCUSSION TITLE
Have anyone used daisy ui with pure typescript project?

### DISCUSSION THREAD

**Discussion by aloksharma1** [saadeghi/daisyui#2435] [Q&A]
Hi All,
i am trying to build a platform agnostic library and for ui part i wanted to use tailwind and stumbled upon daisyui. Anyways in summary i was able to setup everything but when i build & run my project the generated css have no definition for grids/flex/buttons etc, there is no error but the styling is just missing.
i am using webpack+es 6+typescript and now i am at my wits end on where i am going wrong with this thing. i am just in initial phase so if anyone have a better working idea i am all set for it (i just want my output libs to be platform agnostic & standalone).

thanks

note: if i use `npx tailwindcss -i ./src/styles/common.css -o .//output.css --watch`  i get the correct css, but in a dynamic project using lit html and templates in ts files, i dont think its a feasible solution because in your example you are not generating css separately.

**Comment by saadeghi** [comment]
Sharing a reproduction repo would be helpful.

**Reply by aloksharma1** [reply]
hi thanks for the reply, as per your request i have a made a repo here : https://github.com/aloksharma1/daisyui-typescript-issue

i am getting a warning when i build or serve the project , which is related to this issue i think:
```
warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
```

but you can check in the repo i have already defined everything as per tailwind and daisyui documentation.

awaiting your reply.

**Reply by aloksharma1** [reply]
Hello, dont want to rush you or anything, but were you able to look into this issue?
